packages/http-client-csharp/emitter/src/lib/client-model-builder.ts

```diff
@@ -168,22 +167,21 @@ export function createModel(sdkContext: CSharpEmitterContext): CodeModel {
           }
         : fromSdkType(sdkContext, parameter.type); // TODO: consolidate with converter.fromSdkEndpointType
       parameters.push({
-        Name: parameter.name,
-        NameInRequest: parameter.serializedName,
-        Summary: parameter.summary,
-        Doc: parameter.doc,
+        name: parameter.name,
+        nameInRequest: parameter.serializedName,
+        summary: parameter.summary,
+        doc: parameter.doc,
         // TODO: we should do the magic in generator
-        Type: parameterType,
-        Location: RequestLocation.Uri,
-        IsApiVersion: parameter.isApiVersionParam,
-        IsResourceParameter: false,
```

Comment:
the property `isResourceParameter` is never assigned to `true`.
We have this property because we copied code from `autorest.csharp`.
This property only serves the purpose of backward compatible to swagger inputs, we do not have this concept on typespec.
Therefore here we are removing it.